core/src/main/java/bio/terra/pearl/core/model/site/SiteImageMetadata.java

```diff
@@ -0,0 +1,24 @@
+package bio.terra.pearl.core.model.site;
+
+import bio.terra.pearl.core.model.BaseEntity;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.experimental.SuperBuilder;
+
+/** Class for storing metadata about a portal image.  Does not contain the serialized image data */
+@Getter
+@Setter
+@SuperBuilder
+@NoArgsConstructor
+public class SiteImageMetadata extends BaseEntity {
+    /**
+     * the cleanFileName essentially serves as a stableId -- we call it cleanFileName here to indicate its origin.
+     * and because it is not required to be globally unique like other stableIds -- it just has to be unique within a portal
+     */
+    private String cleanFileName;
```

Comment:
Ah, what was not clear until I read the frontend code is that effectively `cleanFileName`, `version` and `portalShortcode` are a key. That may be worth mentioning here since it makes the field use clearer.
If that's true, can `uploadFileName` vary across different versions?